app/src/main/java/org/simple/clinic/summary/bloodpressures/newbpsummary/NewBloodPressureSummaryViewConfig.kt

```diff
@@ -0,0 +1,5 @@
+package org.simple.clinic.summary.bloodpressures.newbpsummary
+
+data class NewBloodPressureSummaryViewConfig(
+    val numberOfBpsToDisplay: Int = 3
```

Comment:
We try to avoid using default parameters because it hides details. Our convention, especially for configuration objects, is to set the default values when instantiating the instance in the Dagger graph. See `PatientSummaryConfig` for an example.

This is fine for now, and we can make this change in the future PR when adding the view to the screen.

Reply:
Fixed it